web/packages/teleport/src/components/ButtonLockedFeature/ButtonLockedFeature.tsx

```diff
@@ -50,7 +53,9 @@ export function ButtonLockedFeature({
     <ButtonPrimary
       as="a"
       target="blank"
-      href={`${SALES_URL}?${getParams(version, isEnterprise, event)}`}
+      href={`${
+        cfg.isUsageBasedBilling ? UPGRADE_TEAM_URL : UPGRADE_COMMUNITY_URL
+      }?${getParams(version, isEnterprise, event)}`}
```

Comment:
I think defining this above and referencing the variable would look a bit neater

```typescript
const baseUrl = cfg.isUsageBasedBilling ? UPGRADE_TEAM_URL : UPGRADE_COMMUNITY_URL;
const upgradeUrl = `${baseUrl}?${getParams(version, isEnterprise, event)}`;
```